### ISSUE TITLE
Dashboard: Trends Overview

### ISSUE THREAD

**Issue by bastianilso** [med-material/trail-it-analysis#3]
![dashboard-trends-overview](https://user-images.githubusercontent.com/3967945/140946467-142d3c43-3187-4537-9105-83db2cd72508.png)

**Comment by Jules-CLERC** [comment]
**Regarding filter buttons (Patient, Outpatients...) :** 

Does the data change also apply to "Training Programs" ? Or the part "training Programs" only takes into account the dataset with all the sql data ? 

I ask this question because the fact that the buttons are in the part "demographics" doesn't make sense to me (if the change also applies to "Training Programs")...

If the buttons apply to both sections, would you prefer to move the filter buttons above the title "demographics" ? this shows that the buttons are not only useful in a section but well on the whole page

**Comment by bastianilso** [comment]
Hi Jules, that's a good point - in fact, maybe these categories can apply onto the whole page. What do you think?

In that case, we should definitely move the button toolbar to the very top of the page, rather than under the "Demographics" headline. It should also apply to the aggregate numbers.

**Comment by Jules-CLERC** [comment]
I think it would be a good idea to apply the changes to both sections.
On the other hand, I don't know if it really has an interest to apply the modifications on the aggregate numbers 🤔

**Comment by Jules-CLERC** [comment]
<img width="1347" alt="image" src="https://user-images.githubusercontent.com/48790000/142211011-a80606a4-1c78-41a1-98ea-84e2b178db34.png">
<img width="1347" alt="image" src="https://user-images.githubusercontent.com/48790000/142211093-1b0b4bd3-93da-4e8c-b3e9-13701be55b44.png">
What do you think of this dashboard ? Do you have any suggestions ?

**Comment by bastianilso** [comment]
Looks good Jules, functionality-wise, this dashboard is covering most of what we want to see. What we need to do here next, will be a visual overhaul, the dashboard feels more balanced. Check this quick mockup below:

 * use similar width/height fr charts as the play context table. If the width/height becomes too small for the chart labels, increase width/height of the play context table a little by introducing more spacing between columns/rows. Font sizes should remain as they are (I just cheated by scaling/zooming the charts in my mockup, but ignore that).
 * the game type table only has two rows so maybe we can just flow this information horizontally (instead of a table).
 * Let's add a divider (hr) between the stats and the filter, and add an additional filter for 'Game Type'.
 * Under each headline, let's have a brief sentence where we describe medians, based on people's selections.

![image](https://user-images.githubusercontent.com/3967945/142375958-136033fb-ea9a-4386-8b8a-602de53cf7d2.png)

**Comment by Jules-CLERC** [comment]
Hi bastian,

About the filter buttons (Filter players and filter game type), what are the default values ?

**Comment by bastianilso** [comment]
Hi Jules,

Filter players: Only stroke patients + outpatients
Filter game type: Both

**Comment by bastianilso** [comment]
![dashboard-trends-overview](https://user-images.githubusercontent.com/3967945/142868282-36884e0c-b8be-442d-9542-2201e2495287.png)

**Comment by Jules-CLERC** [comment]
<img width="1007" alt="image" src="https://user-images.githubusercontent.com/48790000/143213519-a283d702-7205-40d8-b96c-2981df08fa97.png">

Hi Bastian, 
about the timeline, The graph currently looks like this. do you have any advice or things you want to change ?
For example, do you want to display the players having only one point (ie having only played on one date).

**Comment by bastianilso** [comment]
Hi Jules, excellent, very cool. 🍹
I think we want a slider underneath, which provide easy horizontal zooming of the data. see a reference implementation here:
https://plotly.com/r/range-slider/

Also, is each line a player or is each line a "player-gametype" (e.g. if someone played both modes, do they display as 1 or 2 lines?) I think the expected behavior we want, is that each line = a player.

**Comment by Jules-CLERC** [comment]
<img width="1007" alt="image" src="https://user-images.githubusercontent.com/48790000/143216120-a89c0009-1d85-49ef-81ca-afb57045e9c1.png">

I will implement it immediately !
Each line corresponds to a player (so 1 line by player in every case) 🦭.

**Comment by Jules-CLERC** [comment]
<img width="1007" alt="image" src="https://user-images.githubusercontent.com/48790000/143260352-3913afbc-e433-4cfd-943b-a6de8b618a45.png">

is that what you expected?

**Comment by bastianilso** [comment]
Yes. I wonder what options do we have for customizing it? Couls you check?

**Comment by Jules-CLERC** [comment]
I can change the height (thickness).
I can change the style (for example the background color)
I can change the y axis range (for example if you want the rangeslider to display only the first 10 players)

I think I've summarized the [documentation](https://plotly.com/r/reference/#layout-xaxis-rangeslider) for rangeslider.

**Comment by bastianilso** [comment]
Thanks jules, this was the link I was looking for. Can you apply a 1px black border around it (or same pixel width as the plot uses for x/y-axis).  Is 'type' set to 'date' ?

**Comment by Jules-CLERC** [comment]
<img width="1015" alt="image" src="https://user-images.githubusercontent.com/48790000/143838359-f06bb472-1461-4ec3-b72d-1747fd009662.png">

Yes the 'type' is set to 'date'. Does it suit you? 🦖

**Comment by bastianilso** [comment]
perfect, thanks jules! I will keep the issue open, just in case we add more items to it in the future.
